src/canvastools/ts/CanvasTools/Interface/IEventDescriptor.ts

```diff
@@ -3,7 +3,7 @@
  * @remarks Used as an array type to specify a collection of listeners that should
  * added to each of the elements in another array.
  */
-export interface IEventDescriptor {
+ export interface IEventDescriptor<T extends PointerEvent | MouseEvent | KeyboardEvent | WheelEvent> {
```

Comment:
Do we have tslint/eslint rules that would have caught this?

Reply:
There are rules, but no enforcement. TSlint was warning me about it, so I fixed it. I'm not really focusing on fixing and refactoring primarily, so I'm keeping them as opportunistic updates at the moment. Trying to avoid changing too many contracts/interfaces which would cascade into lots of code changes spread around.

Reply:
Got it. Definitely fine to defer it from this PR, but we may want to turn on stricter lint rules maybe warning as error, in future.